src/TodoListItem.jsx

```diff
@@ -1,6 +1,6 @@
 function TodoListItem({ id, todo, onRemoveTodo }) {
   return (
-    <li key={id}>
+    <li>
```

Comment:
Your `key` property that was removed here should be placed in the `map` callback where this component is rendered, rather than on the component itself.

You can use `key={item.id}` on your `TodoListItem` in your `TodoList.jsx` file.

Once you do that, you can look and see that the console warning about the `key` prop will be gone.

Reply:
You can read more if interested in this section of the React docs:
[https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key](https://react.dev/learn/rendering-lists#keeping-list-items-in-order-with-key)